triggers/service/auth/src/main/scala/com/daml/auth/middleware/oauth2/Config.scala

```diff
@@ -36,13 +38,16 @@ case class Config(
 )
 
 object Config {
+  val DefaultHttpPort: Int = 3000
```

Comment:
any particular reason to choose 3000? I don’t really understand any of our default port numbers so no issue with that just curious.

Reply:
No, no particular reason. This just happened to be the port number used for the oauth middleware in the docs examples.